public/docs/ts/latest/guide/style-guide.jade

```diff
@@ -1303,7 +1299,7 @@ a(href="#toc") Back to top
 
 .s-rule.do
   :marked
-    **Do** put common components, directives and pipes that will be used throughout the application by other feature modules in the `SharedModule`, where those assets are expected to share a new instance of themselves (not singletons).
+    **Do** put common components, directives, and pipes that will be used throughout the application by other feature modules in the `SharedModule`, where those assets are expected to share a new instance of themselves (not singletons).
```

Comment:
Please rewrite as follows:

**Do** declare components, directives, and pipes in a shared module when those items will be re-used and referenced by the components declared in other feature modules.

**Consider** using the name `SharedModule`, when the contents of a shared module are referenced across the entire application.

**Do not** provide services in shared modules. Services are usually singletons that are provided once for the entire application or in a particular feature module.